app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -52,7 +52,7 @@ def self.raw_connect(server, port, username, password, service = "Service")
     Ovirt.const_get(service).new(params)
   end
 
-  def connect(options = {})
+  def other_connect(options = {})
```

Comment:
I don't understand the need for the method.  If you look at the other service providers (like Amazon)...everything ends up at raw_connect where the decision is made.  I can see *there* having a metrics_connect and default_connect, but not at this level.